api/cds.proto

```diff
@@ -161,24 +161,20 @@ message Cluster {
   //   verification.
   UpstreamTlsContext tls_context = 11;
 
-  oneof protocol_options {
-    // [#not-implemented-hide:]
-    TcpProtocolOptions tcp_protocol_options = 12;
-
-    // Additional options when handling HTTP1 requests.
-    Http1ProtocolOptions http_protocol_options = 13;
-
-    // Even if default HTTP2 protocol options are desired, this field must be
-    // set so that Envoy will assume that the upstream supports HTTP/2 when
-    // making new HTTP connection pool connections. Currently, Envoy only
-    // supports prior knowledge for upstream connections. Even if TLS is used
-    // with ALPN, `http2_protocol_options` must be specified. As an aside this allows HTTP/2
-    // connections to happen over plain text.
-    Http2ProtocolOptions http2_protocol_options = 14;
-
-    // [#not-implemented-hide:]
-    GrpcProtocolOptions grpc_protocol_options = 15;
-  }
+  // deleted = 12;
+
+  // Additional options when handling HTTP1 requests.
+  Http1ProtocolOptions http_protocol_options = 13;
+
+  // Even if default HTTP2 protocol options are desired, this field must be
+  // set so that Envoy will assume that the upstream supports HTTP/2 when
+  // making new HTTP connection pool connections. Currently, Envoy only
+  // supports prior knowledge for upstream connections. Even if TLS is used
+  // with ALPN, `http2_protocol_options` must be specified. As an aside this allows HTTP/2
+  // connections to happen over plain text.
+  Http2ProtocolOptions http2_protocol_options = 14;
+
+  // deleted = 15;
```

Comment:
Use proto language syntax as follows. If you are not familiar, please see proto spec.

```
reserved 13, 15;
```

Reply:
Thanks, used the reserved keyword for 12 and 15 separately.